src/plot.jl

```diff
@@ -8,12 +8,13 @@
 
     xguide --> "Strategy"
     yguide --> "round"
+    colorbar_title --> "score"
     seriestype --> :heatmap
     seriescolor --> :RdYlGn_10
 
     ystrings = reverse(string.(rounds))
 
-    xlabels = (1:size(scores, 2), strategies)
+    xlabels = (1:size(scores, 2), last.(split.(string.(strategies), ".")))
```

Comment:
It works I guess, but it's a bit gross.  Did you figure out what the issue was?

Reply:
Not quite, but I will continue to look into this after the MR. This also happens if I don't export Devil or RandomStrategy.